include/hip/hip_runtime_api.h

```diff
@@ -9575,7 +9575,6 @@ return hipOccupancyMaxPotentialBlockSize(gridSize, blockSize,(hipFunction_t)kern
  * @brief Launches a device function
  *
  * @ingroup Execution
- * @ingroup ModuleCooperativeG
```

Comment:
This function needs to be in two module. I know, we get warning, however at least it appears in both section. Please revert this change.

Reply:
Done.